cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -572,16 +670,39 @@ func StartControllers(ctx context.Context, controllerCtx ControllerContext, star
 	//   so we cannot rely on it yet to add the name
 	// - it allows distinguishing between log entries emitted by the controller
 	//   and those emitted for it - this is a bit debatable and could be revised.
-	for controllerName, initFn := range controllers {
-		if !controllerCtx.IsControllerEnabled(controllerName) {
+	for controllerName, regController := range registrableControllers {
```

Comment:
should I make a function from a body of this for so it can be reused above for `ServiceAccountTokenController`?

Reply:
Smaller functions are better.

Reply:
the size most likely won't be that much changed, but I think the biggest benefit would be the deduplication and price small indirection

Reply:
deduplicated